CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Revision history for nix-thunk
 
+## 0.6.0.0
+
+* [#33](https://github.com/obsidiansystems/nix-thunk/pull/33) Fix an
```

Comment:
This should be 34 instead of 33, right?

Reply:
... Right, thanks :smile: apparently I counted wrong when guessing the PR number